tools/reviewer/job/ReviewerJob.java

```diff
@@ -59,19 +63,16 @@
   private Lazy<LocalServer> lazyLocalServer;
   private Lazy<HeadUpdater> lazyHeadUpdater;
 
-  @FlagDesc(name = "service_account_json", description = "", required = true)
-  public static Flag<String> serviceAccountJson = Flag.create("");
-
-  @FlagDesc(name = "repo_url", description = "", required = false)
-  public static Flag<String> repoUrl = Flag.create("");
-
   @Inject
   public ReviewerJob(
-      TaskExecutor taskExecutor,
       InitCommand initCommand,
       Lazy<LocalServer> lazyLocalServer,
```

Comment:
@oferb why are `LocalServer` and `HeadUpdater` `Lazy<>`?

Reply:
Good point. Forgot to document it. Added this:
```
// These are lazy because constructing them requires us to be in a base folder.
// That base folder is only set up in run() using initCommand. We also assumme we're inside it.
```